client-src/overlay.js

```diff
@@ -129,12 +142,10 @@ const createOverlay = (options) => {
     applyStyle(iframeContainerElement, iframeStyle);
```

Comment:
Intentionally keep iframe styling to use inline style because we can't scope CSS for it.